server/app/assets/javascripts/preview.ts

```diff
@@ -252,6 +252,7 @@ class PreviewController {
         PreviewController.QUESTION_HELP_TEXT_SELECTOR,
       )
       if (contentParent) {
+        contentParent.classList.add('mb-4')
```

Comment:
Yup, it adds margin when help text is present. Otherwise the help text is too close to the other divs. We originally added mb-4 server-side to the help-text container but it was rendering empty space when there wasn't help-text.